src/components/Products/GridItem/GridItem.js

```diff
@@ -15,110 +15,117 @@ import { getIsFavoriteStatusFromProducts } from 'store/products/selectors';
-    : 'add to favorites';
+    const favoriteInfoText = isFavoriteStatus
+        ? 'remove from favorites'
+        : 'add to favorites';
 
-  return (
-    <>
-      <li className="gridItem-container" onClick={redirectToDetailPage}>
-        <div className="gridItem">
-          <div className="gridItem__img-wrap">
-            <img src={imageUrl} />
-          </div>
-          <div className="gridItem__description">
-            <p className="gridItem__text">{name}</p>
-            <p className="gridItem__text">${price / 100}</p>
-          </div>
-          <div className="gridItem__first-icon">
-            <ButtonRound
-              icon={iconFav}
-              onClick={favoriteToggleHandler}
-              info={favoriteInfoText}
-              className={favoriteClasses}
-            />
-          </div>
-          <div className="gridItem__show-icon">
-            <ButtonRound
-              icon={iconDetail}
-              onClick={showDetailModalHandler}
-              info="show details"
-              className="button-round"
-            />
-          </div>
-          <div className="gridItem__show-button">
-            <ButtonLong
-              title="Add to cart"
-              onClick={addToCartHandler}
-              icon={iconBtn}
-            />
-          </div>
-        </div>
-      </li>
-      {isDetailModalVisible && (
-        <Modal
-          element={
-            <ShowProductDetail
-              product={{ imageUrl, name, price, id, description, inStock }}
-            />
-          }
-          onClick={closeDetailsModalHandler}
-        />
-      )}
-    </>
-  );
+    return (
+        <>
+            <li className="gridItem-container" onClick={redirectToDetailPage}>
+                <div className="gridItem">
+                    <div className="gridItem__img-wrap">
+                        <img src={imageUrl} />
+                    </div>
+                    <div className="gridItem__description">
+                        <p className="gridItem__text">{name}</p>
+                        <p className="gridItem__text">${price / 100}</p>
+                    </div>
+                    <div className="gridItem__first-icon">
+                        <ButtonRound
+                            icon={iconFav}
+                            onClick={favoriteToggleHandler}
+                            info={favoriteInfoText}
+                            className={favoriteClasses}
+                        />
+                    </div>
+                    <div className="gridItem__show-icon">
+                        <ButtonRound
+                            icon={iconDetail}
+                            onClick={showDetailModalHandler}
+                            info="show details"
+                            className="button-round"
+                        />
+                    </div>
+                    <div className="gridItem__show-button">
+                        <ButtonLong
```

Comment:
This:
![Screenshot 2023-03-15 at 21 38 58](https://user-images.githubusercontent.com/2543423/225437030-0bd082f2-c87f-4cde-b562-958f1ed842f6.png)

Should become:
![Screenshot 2023-03-15 at 21 41 15](https://user-images.githubusercontent.com/2543423/225437098-03ed2fe9-00b8-4501-8c6b-83c0ece8e880.png)

Reply:
I did my best 😄